components/Tabs/src/private/MDCItemBarCell.m

```diff
@@ -563,7 +563,7 @@ - (void)updateDisplayedImage {
 }
 
 - (void)updateDisplayedTitle {
-  if (!_titleLabel.hidden) {
+  if (_style.shouldDisplayTitle) {
     _titleLabel.text = [[self class] displayedTitleForTitle:_title style:_style];
```

Comment:
If we are using the .hidden property, can we drop the if clause and always set _titleLabel.text?

Reply:
Done.